api/src/main/java/io/strimzi/api/kafka/model/Kafka.java

```diff
@@ -43,7 +39,25 @@
                 ),
                 group = Kafka.RESOURCE_GROUP,
                 scope = Kafka.SCOPE,
-                version = Kafka.VERSION
+                version = Kafka.V1BETA1,
+                versions = {
+                        @Crd.Spec.Version(name = Kafka.V1BETA1, served = true, storage = true),
+                        @Crd.Spec.Version(name = Kafka.V1ALPHA1, served = true, storage = false)
+                },
+                additionalPrinterColumns = {
+                        @Crd.Spec.AdditionalPrinterColumn(
+                                name = "Kafka scale",
```

Comment:
Why "scale"? Isn't it better just "replicas" or "brokers" (for Kafka) and "nodes" (for Zookeeper)?